src/vite-env.d.ts

```
@@ -1 +1,34 @@
+  login: '/api/auth/login',
+  
+  // Upload
+  upload: '/api/upload',
+  getUploads: '/api/uploads',
+  
+  // Chat
+  ask: '/api/ask',
+  
+  // User
+  getMe: '/api/me'
+};
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)